main.go

```diff
@@ -448,3 +403,153 @@ func Main(args *CLIArgs) error {
 
 	return nil
 }
+
+func generateOutput(packages string, outputFormat string, outputType string, mainMods map[string]struct{}, mainLibPkgs []string, mainCmdPkgs []string, modNames []string, modLicenses map[string]map[detectlicense.License]struct{}, modInfos map[string]*golist.Module, goVersion string) (*bytes.Buffer, error) {
```

Comment:
Yes, that's really long.

I think the function is taking too many parameters and with some refactoring maybe it can be made cleaner but I didn't want to spend too much time doing that (at least not for now).

I'll split the line so it's easier to read